solr/core/src/java/org/apache/solr/cli/SolrCLI.java

```diff
@@ -792,11 +792,10 @@ public static String normalizeSolrUrl(CommandLine cli) throws Exception {
 
           String firstLiveNode = liveNodes.iterator().next();
           solrUrl = ZkStateReader.from(cloudSolrClient).getBaseUrlForNodeName(firstLiveNode);
-          // solrUrl = normalizeSolrUrl(solrUrl, false);
+          solrUrl = normalizeSolrUrl(solrUrl, false);
         }
       }
     }
-    solrUrl = normalizeSolrUrl(solrUrl, false);
```

Comment:
Not sure about this - whether all use of this. Will caller always expect to get `/solr` stripped after calling this? Guess they do, but started by reverting some changes to see if I could get to previous state..